### DISCUSSION TITLE
求助！！很紧急！！

### DISCUSSION THREAD

**Discussion by en-461** [share121/inter-knot#320] [其他]
![Uploading Image_1723128891561.jpg…]()
如图
昨天想试试新买的萝卜好不好用，就进了零号空洞。在我闲逛的时候，我突然看见了一只猫！她好可爱呜呜呜，于是就没忍住上去摸了一下，结果她一下子暴起给我挠成了这样，空洞里的猫应该不会带有什么奇怪的病毒吧呜呜呜，不会有什么生命危险吧呜呜，求建议

**Comment by en-461** [comment]
具体如图
![Uploading Image_1723128891561.jpg…]()

**Comment by godfather-max** [comment]
先做十个俯卧撑压压惊
